src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,25 @@
+module Crocodealer.Config
+  ( LabelRule
+  , Label
```

Comment:
I guess you would need to import all of the constructors and fields for each data type here.

Reply:
I am not sure if I undestood this one. Can you elaborate more please Veronika?